src/lib/currentRoutePath.ts

```diff
@@ -0,0 +1,13 @@
+import { parse, join } from "path";
+
+/**
+ * Returns path to current view.
```

Comment:
I'd avoid "view". Its not MVC architecture and in frontend-centric apps it has very different meanings

Maybe Route is a good name